packages/open-payments/src/client/token.test.ts

```diff
@@ -85,4 +140,55 @@ describe('token', (): void => {
       scope.done()
     })
   })
+
+  describe('revokeToken', (): void => {
+    test('returns accessToken if passes validation', async (): Promise<void> => {
```

Comment:
```suggestion
    test('returns undefined if successfully revokes token', async (): Promise<void> => {
```
or something along those lines